packaging/helm/skipper-aws/templates/external-dns/aws-iam-role.yaml

```
@@ -0,0 +1,9 @@
+{{ if .Values.kube_aws_iam_controller }}
+apiVersion: zalando.org/v1
+kind: AWSIAMRole
+metadata:
+  name: external-dns-aws-iam-credentials
```

Each Helm install gets a release name (either autogenarated or provided by the user) that allows multiple instances of a single chart to be installed. It's good practice to use this name in the name of resources.
```suggestion
  name: {{ .Release.Name }}-external-dns-aws-iam-credentials
```

Docs: https://helm.sh/docs/chart_template_guide/builtin_objects/